src/crypto/PKCS11Signer.cpp

```diff
@@ -327,7 +327,8 @@ vector<unsigned char> PKCS11Signer::sign(const string &method, const vector<unsi
         }
         switch(rv)
         {
-        case CKR_OK: break;
```

Comment:
This leads to bad readability and is error-prone. The fallthrough can stay, but should be made more explicit.